scala/private/phases/phase_compile.bzl

```diff
@@ -309,16 +309,26 @@ def _create_scala_compilation_provider(ctx, ijar, source_jar, deps_providers):
     )
 
 def _pack_source_jar(ctx, scala_srcs, in_srcjars):
```

Comment:
was confused a bit at the beginning as `in_srcjars` implies a bool, but looks like it's meant to be a list of src jars.

so nit possibly to rename it `scala_src_jars`. feel free to do so as a follow up.

Reply:
Agreed, though we should rename to `input_srcjars` imo (as followup).

Reply:
sgtm. thanks!

Reply:
Too lazy for followup. Renamed it and also changed the `.-src.jar` naming into `-src.jar`.